Misc/NEWS.d/next/Library/2024-12-06-17-28-55.gh-issue-127610.ctv_NP.rst

```diff
@@ -0,0 +1,2 @@
+Added validation for more than one var-positional or
+var-keyword parameters in :class:`inspect.Signature`.
```

Comment:
If you want you can add "Patch by Maxim Ageev." as well

Reply:
Thanks for the information, I added